site/src/components/Abbr/Abbr.tsx

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
There are a lot of comments in here for a relatively small component. Are there any we can shorten?